### ISSUE TITLE
Floating calculator feature

### ISSUE THREAD

**Issue by VasilisKos** [forzzzzz/Calculator-You#14]
A floating calculator over other apps, when tapping the home button, would be very convenient when using at the same time other apps (e.g. web browsers).
The Xiaomi Calculator is the only one I've seen with such a feature.

**Comment by PhilippKosarev** [comment]
Hey, I know most Android distros have an "Open in pop-up view" option (OneUI and MiUI have it afaik), is that available on your system?

**Comment by VasilisKos** [comment]
> Hey, I know most Android distros have an "Open in pop-up view" option (OneUI and MiUI have it afaik), is that available on your system?

No. I have a Nokia X20 with pure Android. So, it doesn't have such an option.

**Comment by PhilippKosarev** [comment]
I'm sorry for your loss

**Comment by medicamin** [comment]
I suggest opening a floating version via quick settings tile.
